src/main/java/ovh/akio/hmu/GameUnpacker.java

```diff
@@ -39,7 +47,7 @@ public File getWorkspace() {
 
     public File getUnpackingOutput() {
 
-        return Utils.asLocalDirectory("extracted", this.game.getShortName());
+        return Utils.asLocalDirectory(outputDir, this.game.getShortName());
```

Comment:
```suggestion
        return Utils.asLocalDirectory(this.outputDir, this.game.getShortName());
```

Same as the previous comment